packages/cli/odahuflow/cli/parsers/local/packaging.py

```diff
@@ -60,6 +62,136 @@ def cleanup_containers():
     cleanup_packaging_docker_containers()
 
 
+def fetch_local_entities(manifest_file, manifest_dir):
```

Comment:
```suggestion
def fetch_local_entities(manifest_file: List[str], manifest_dir: List[str]) -> List[OdahuflowCloudResourceUpdatePair]:
```

+ `from typing import List` in the top